std_msgs/Header header
sensor_msgs/BatteryState batteryPackA
sensor_msgs/BatteryState batteryPackB
float64 batteryPacksTotalCurrent

uint8 batteryCell1Thermocouple
uint8 batteryCell2Thermocouple
uint8 batteryCell3Thermocouple
uint8 batteryCell4Thermocouple
uint8 batteryCell5Thermocouple
uint8 batteryCell6Thermocouple
uint8 batteryCell7Thermocouple
uint8 batteryCell8Thermocouple

uint8 systemThermocouple1
